lib/ddtrace/runtime/cgroup.rb

```diff
@@ -1,5 +1,10 @@
 require 'ddtrace/ext/runtime'
 
+# TODO: suggested namespace
+# Datadog::Tracing::Environment::Ruby::GC
+# Datadog::Tracing::Environment::Socket
+# Datadog::Tracing::Runtime/Engine
```

Comment:
I don't quite understand this TODO...?

Reply:
So the name `Datadog::Runtime` is taken today by a module for stuff like `Datadog::Runtime::CGroup`, but the "runtime" introduce in this PR is very clearly described as "the" runtime for `ddtrace`.

I'd like to reclaim the `Datadog::Runtime` namespace for the runtime class, but that will require moving stuff like `Datadog::Runtime::CGroup` to a different namespace.

Reply:
That's an excellent thing to copy paste as a comment there ;)

Reply:
I don't think I'll submit a non-draft PR without actually addressing the namespace conflicts (thus not needing the TODO).

I guess this comment was a note to myself, pretend you didn't see it 🪄👋.

Reply:
# **~~👀~~**